examples/webgl_materials_blending.html

```diff
@@ -98,6 +88,8 @@
 						material.transparent = true;
 						material.blending = blending.constant;
 
+						if ( material.blending === THREE.SubtractiveBlending ) material.opacity = 0;
```

Comment:
Is this temporary?

Reply:
It is required now since there is an alpha channel. Subtractive blending subtracts the source colors and alpha from the destination. With an opacity value of 1, the final alpha gets zero which means the HTML background will bleed through.

Reply:
Ah, interesting...
So we should change the `SubstractiveBlending` to not affect alpha?  At least that's how Photoshop behaves...

Reply:
Fixed! Not sure how to change the premultiplied alpha case though.

Reply:
BTW: BabylonJS does not modulate subtractive blending by alpha values. So it seems the new approach is more correct than before.

Reply:
I've premultiplied alpha case exactly like the non premultiplied alpha for now.

Reply:
Hmm, just noticed Babylon does `gl.ONE` `gl.ONE` for `srcAlpha` and `dstAlpha` ([doc](https://developer.mozilla.org/en-US/docs/Web/API/WebGLRenderingContext/blendFuncSeparate)):

https://github.com/BabylonJS/Babylon.js/blob/c10fa15042293fa12f24ee44f7cd0a1a18343f03/src/Engines/Extensions/engine.alpha.ts#L78

Reply:
Um, but this will add both alpha values together in our case. The combination of `gl.ZERO` and `gl.ONE`  retains alpha value in the destination. Maybe they don't use `FUNC_ADD` like `three`?

Reply:
> Maybe they don't use `FUNC_ADD` like `three`?

Ah. Interesting...

Reply:
Yeah, I think that makes sense.
This stuff is harder to wrap your head around than PBR equations... 😬

Reply:
Indeed!. It would be also interesting to know how often the more special blending like `SubtractiveBlending` are actually used in production 🤓 .